### PR TITLE
Fixes syringe gun and bee injection reactions

### LINKED ISSUE #7697: Nano machines and Odysseus
Server: Sybil

Revision:2015-02-12
0d211471b872fb12bf93d1e5d27a33666ca1c4d9 
Problem Description: When you shoot a person with a syringe from an Odysseus that contains nano machines, it doesn't actually inject the mob with nano machines like it should be. Because when you shoot a syringe containing nano machines, and don't actually hit anything, you can get out of the mecha and stab people with it to make it work like it's supposed to by turning them into borgs after about a minute.

What did you expect to happen: Get a roburger, get an oddy, grind the roburger and scan it with the oddy, shoot people with nano machines from the safety of your oddy and watch as the station is turned to silicons

What happened instead: Shot people in escape with syrynges containing nano machines, waited till the end of the round for anybody to change, but nobody did, right after the shuttle docked I got out to test if the nenomachienes even worked, stabbed with my self with it, and sure enough I got the message saying beep boop that generally means you've been infected with the virus.

Why is this bad/What are the consequences:

Steps to reproduce the problem: The most important section. Review everything you did leading up to causing the issue.
Do xenobio.

Get silver slimes, dark purple slimes, purple slimes, cerulean slimes, regular grey slimes, and metal slimes.

Use all the steroids on your silver slimes so you can use it a ton of times.

Repeat until you spawn a roburger via the plasma reaction with silver slime extract.

Grind the roburger in your grinder.

Use the metal slimes to build an Odysseus

Scan the beaker containing the remains of the roburger with the syringe gun on the Odysseus.

Load the syringe gun with syringes from your box in xenobio.

Begin production of nano machines in your oddy on the reagents list of the syringe gun in the Odysseus's status menu.

Shoot anyone, watch as nothing happens and you've just wasted what is probably three hours of work for nothing.

Shoot the ground, or anything, just don't hit anything.

Get out of mech.

Grab syringe.

Stab self with said syringe.

Watch as you become borg.

 which gamemode was it?
Just standard Traitor

Was anything else out of the ordinary happening?: If you inject yourself with a syringe from said oddy with nano machines, instead of getting hit with they syringe, you WILL start to turn into a borg.

### LINKED ISSUE #16455: Nanomachine reagent bees don't work properly?
Last night a traitor traded his tcs for 15u of nano machines in which he used it on bees. Bees were stinging people but nobody was being infected with nano machines.

### PR DESCRIPTION
**Pull request by Shadowlight213**
Fixes #16455
Fixes # #7697

### CODE REVIEW

**Comment by Core0verload** [comment]
Doesn't actually fix #7697, because Oddy syringe gun uses dark magic, not syringe projectiles. But I'll fix it.

**Comment by Shadowlight213** [comment]
Now it does!

File: code/modules/mob/living/simple_animal/hostile/bees.dm [hunk 1]
```diff
@@ -144,7 +144,7 @@
 	else
 		if(beegent && isliving(target))
 			var/mob/living/L = target
-			beegent.reaction_mob(L, TOUCH)
+			beegent.reaction_mob(L, INJECT)
```
Comment (Fox-McCloud) [inline code/modules/mob/living/simple_animal/hostile/bees.dm]: RIP synth flesh bees.

Comment (TechnoAlchemisto) [inline code/modules/mob/living/simple_animal/hostile/bees.dm]: good thing some1 readded those trek chems

Comment (Fox-McCloud) [inline code/modules/mob/living/simple_animal/hostile/bees.dm]: Which you'll promptly OD on and die.

Comment (PKPenguin321) [inline code/modules/mob/living/simple_animal/hostile/bees.dm]: worth

Comment (TechnoAlchemisto) [inline code/modules/mob/living/simple_animal/hostile/bees.dm]: keep that up and you're gonna be od'in on these hands my dude

**Pull request merged**